app/gui2/stories/story.css

```diff
@@ -1,6 +1,13 @@
+@import url('../src/assets/font-enso-code.css');
+
+:root {
+  --font-code: 'Enso Code', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Comment:
Why is it necessary to repeat this css here? Shouldn't the declaration from `base.css` already work here?

Reply:
I guess I'll test this, and remove if it is unnecessary